sharepoint/3.1/modules/ROOT/pages/sharepoint-connector-reference.adoc

```diff
@@ -1353,266 +1398,261 @@ Executes an HTTP GET against the SharePoint API (POST if body is a blob).
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS | A time unit that qualifies the maxIdleTime attribute. | |
+** DAYS | A time unit that qualifies the maxIdleTime attribute |  |
 |===
 
-[[AttachmentInformation]]
-=== Attachment Information
+[[RedeliveryPolicy]]
+=== Redelivery Policy
 
-[%header,cols="20s,20a,35a,20a,5a"]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| File Content Stream a| Binary | | |
-| File Name a| String | | |
-| Item Id a| String | | |
+| Max Redelivery Count a| Number | The maximum number of times a message can be redelivered and processed unsuccessfully before triggering a process-failed message |  |
+| Use Secure Hash a| Boolean | Whether to use a secure hash algorithm to identify a redelivered message. |  |
+| Message Digest Algorithm a| String | The secure hashing algorithm to use.  | SHA-256 |
+| Id Expression a| String | Defines one or more expressions to use to determine when a message has been redelivered. This property can be set only if useSecureHash is false. |  |
+| Object Store a| Object Store | The object store where the redelivery counter for each message is stored. |  |
 |===
 
-[[SharePointFile]]
+[[DeletedObject]]
```

Comment:
Everything should have descriptions, but this is probably a problem with the source. Let's make a note of all these missing descriptions in our spreadsheet in case it's a bug. If it's not a bug, we need to add all the descriptions in the javadoc.